src/hl7v2/schema.clj

```diff
@@ -4,7 +4,8 @@
    [clojure.java.io :as io]
    [clojure.string :as str]
    [clojure.walk :refer [postwalk]]
-   [hl7v2.complex :refer [clean]])
+   [hl7v2.complex :refer [clean]]
+   [hl7v2.config :refer [config]])
   (:import
    (java.text Normalizer Normalizer$Form)))
```

Comment:
como tal qué sería java.text? una librería de java?

Reply:
Clojure is a hosted language that runs on the Java virtual machine(JVM), this is a pragmatic decision made to gain access to the huge Java ecosystem and its standard library. In this case, java.text is part of the standard library, aka jdk. `java.text` is a package of java, see the javadocs(https://docs.oracle.com/javase/8/docs/api/java/text/package-summary.html)